Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.xaml

```diff
@@ -1,7 +1,7 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                     xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls"
-                    xmlns:wrapPanel="using:Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel">
+                    xmlns:wrapPanel ="using:Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel">
```

Comment:
do we still have the Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel namespace ? I think we agreed on it being a breaking change.. that means WrapPanel goes inside Microsoft.Toolkit.Uwp.UI.Controls namespace